samples/dect/dect_phy/dect_shell/README.rst

```diff
@@ -110,6 +118,18 @@ Examples
 
      dect rssi_scan stop
 
+* Execute longer RSSI measurements on all permitted channels, and instead of default high/low RSSI value based verdict; use subslot count based verdict for BUSY/POSSIBLE/FREE and as a end results, see the verdict for the best channel:
+
+  .. code-block:: console
+
+     dect rssi_scan -c 0 --verdict_type_count -t 3000 -a
+
+* Execute longer RSSI measurements on specific channel, and instead of default high/low RSSI value based verdict; use subslot count based verdict for BUSY/POSSIBLE/FREE. Additionally, print BUSY/POSSIBLE measurements:
```

Comment:
```suggestion
* Execute longer RSSI measurements on specific channel, and instead of default high/low RSSI value-based verdict, use subslot count-based verdict for BUSY/POSSIBLE/FREE.
  Additionally, print BUSY/POSSIBLE measurements:
```